tests/serverpod_test_server/test_e2e/auth_core_test.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/config.dart';
+import 'package:serverpod_test_server/test_util/test_key_manager.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+void main() {
```

Comment:
Question: How are these tests differentiated between the tests in: `tests/serverpod_test_server/test_integration/auth_core/auth_header_test.dart`?

Reply:
They are logically the same! However this one is on the HTTP request level (bypassing all client code) and the other actually uses the client code but bypasses the top layer in order to force set the desired request shape. 

The other one was the first one I wrote and I kept it for now - should we remove it? As was mentioned in a discussion, that test could be imaged to break upon some future client code change.

Sorry should have explained this in the PR.

Reply:
I've removed the other test.